packages/web3-core-promievent/README.md

```diff
@@ -10,36 +10,41 @@ Please read the [documentation][docs] for more.
 
 ## Installation
 
-### Node.js
+You can install the package either using [NPM](https://www.npmjs.com/package/web3-core-promievent) or using [Yarn](https://yarnpkg.com/package/web3-core-promievent)
+
+### Using NPM
 
 ```bash
 npm install web3-core-promievent
 ```
 
+### Using Yarn
+
+```bash
+yarn add web3-core-promievent
+```
+
 ## Usage
 
 ```js
-const Web3PromiEvent = require('web3-core-promievent');
+const Web3PromiEvent = require("web3-core-promievent");
 
-const myFunc = function(){
+const myFunc = function () {
     const promiEvent = Web3PromiEvent();
-    
-    setTimeout(function() {
-        promiEvent.eventEmitter.emit('done', 'Hello!');
-        promiEvent.resolve('Hello!');
+
+    setTimeout(function () {
+        promiEvent.eventEmitter.emit("done", "Hello!");
```

Comment:
Is this styling change ( and the following ones) intentional or from auto-formatter?

Reply:
This is done by prettier by mistake. I reverted the unintended changes.
Thanks,